cirq-web/cirq_web/bloch_sphere/bloch_sphere.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import webbrowser
+import uuid
+
+from typing import Union
+from pathlib import PosixPath, WindowsPath
+
+from numpy import ndarray
+
+from cirq_web import widget
+
+from cirq.testing import random_superposition
+from cirq.qis import to_valid_state_vector
+from cirq.qis.states import bloch_vector_from_state_vector
+from cirq.protocols import to_json
+
+
+class BlochSphere(widget.Widget):
+    def __init__(
+        self,
+        sphere_radius: int = 5,
+        state_vector: ndarray = to_valid_state_vector([math.sqrt(2) / 2, math.sqrt(2) / 2]),
```

Comment:
We should not give a default state vector, or make it `None` instead.
Also, type should be `cirq.qis.STATE_VECTOR_LIKE`.